homework/index.js

```diff
@@ -36,11 +36,115 @@
       if (err) {
         createAndAppend('div', root, { text: err.message, class: 'alert-error' });
       } else {
-        createAndAppend('pre', root, { text: JSON.stringify(data, null, 2) });
+        // create the basic element tags //
+        // the main header.
+        createAndAppend('h1', root, { text: 'HYF Repositories' });
+
+        // the container
+        createAndAppend('div', root, { id: 'container' });
+        const container = document.getElementById('container');
+
+        // the basic repository information (left aside).
+        createAndAppend('aside', container, {
+          class: 'left-aside box',
+          id: 'basicInfo',
+        });
+        const basicInfo = document.getElementById('basicInfo');
```

Comment:
Same as above. I see that you have followed a similar approach of creating an element and then looking it up on document when you can simply save the returned value of `createAndAppend` and assign it to a variable. Can you please update all instances?